src/domain/functions/questionGenerator/questionGenerator.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+import { StarWarsApi } from '../../../api/StarWarsApi';
+
+import { getRandomInt } from '../../../helpers';
+
+export const generateAnswers = async ({ mode, minId, maxId }) => {
+  const randomIds = [];
+  for (let x = 0; x < 4; x++) {
+    let randomId;
+    do {
+      randomId = getRandomInt({ min: minId, max: maxId });
+    } while (randomIds.includes(randomId));
+    randomIds.push(randomId);
```

Comment:
może udałoby się to uprościć funkcjami Math? np. użyciem Math.random()

Reply:
Całą pętle czy funkcje losującą? Funkcja getRanomInt jest oparta na Math.random tylko zdefiniowana w innym pliku.